spec/models/registration_spec.rb

```diff
@@ -40,10 +40,11 @@
           location = build(:location, :supported)
           registration = build(:registration, zipcode: location.zipcode)
 
-          registration.save
+          saved = registration.save
           user = registration.user
           location = registration.location
 
+          expect(saved).to be true
```

Comment:
This expectation came about specifically to guard against a bug that was uncovered elsewhere.

`Registration` is a form object that composes a few ActiveRecord objects. The last record (the `DonorEnrollment` -- another form object) can return non-truthy values from `#save` which is an acceptable state.

The expectation here assures that the `transaction` block returns a truthy value so that other AR-like flows elsewhere will behave properly.